src/System.Management.Automation/engine/remoting/commands/NewPSSessionOptionCommand.cs

```diff
@@ -29,9 +29,8 @@ public sealed class NewPSSessionOptionCommand : PSCmdlet
         [Parameter]
         public int MaximumRedirection
         {
-            get { return _maximumRedirection.Value; }
-
-            set { _maximumRedirection = value; }
+            get => _maximumRedirection ?? -1;
+            set => _maximumRedirection = value;
```

Comment:
[From docs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/new-pssessionoption?view=powershell-7.1):
> The default value is 5. A value of 0 (zero) prevents all redirection.

Is real default 0? /cc @PaulHigin

Reply:
Default changed to `0`.